lib/builder.js

```diff
@@ -122,7 +123,7 @@ function writeOutputFile(opts, output, basePath) {
   });
 }
 
-exports.inlineSourceMap = function(output) {
+var inlineSourceMaps = exports.inlineSourceMap = function(output) {
```

Comment:
I generally prefer a pattern here of:

```javascript
exports.inlineSourceMaps = inlineSourceMaps;
function inlineSourceMaps(output) {
}
```

Feels ES6y having declarativey exports.

Also we should try and maintain consistency with the plurals?